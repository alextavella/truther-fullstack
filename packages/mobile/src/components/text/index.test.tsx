import { render } from '@testing-library/react-native'

import { Text } from './index'

describe('<Text />', () => {
  test('should render correctly', () => {
    const { getByText } = render(<Text>Welcome!</Text>)
    getByText('Welcome!')
  })

  it('snapshot', async () => {
    const tree = render(<Text>Some text</Text>).toJSON()
    expect(tree).toMatchSnapshot()
  })
})
