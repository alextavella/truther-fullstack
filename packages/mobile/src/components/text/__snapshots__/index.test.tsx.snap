// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Text /> snapshot 1`] = `
<Text
  style={
    [
      {
        "color": "#1A1F24",
        "fontFamily": "DMSans_400Regular",
        "fontSize": 16,
        "fontWeight": "regular",
        "textAlign": "left",
      },
      undefined,
    ]
  }
>
  Some text
</Text>
`;
